src/pool/boxed.rs

```diff
@@ -568,4 +603,57 @@ mod tests {
         assert!(once.is_ok());
         assert!(twice.is_ok());
     }
+
+    #[test]
+    fn test_into_raw_from_raw() {
+        pub struct MyStruct {
+            value: [u8; 64],
+        }
+
+        static DROP_WAS_CALLED: AtomicUsize = AtomicUsize::new(0);
```

Comment:
Why not AtomicBool? 🤔

Reply:
That's a fair point :) lemme update it real quick